spec/integration/filtering_spec.rb

```diff
@@ -66,6 +66,47 @@ def run_rerun_command_for_failing_spec
   end
 
   context "passing a line-number filter" do
+    it "works with different custom runners used in the same process" do
+      class Foo
+        attr_accessor :i
```

Comment:
I know this is a dummy formatter, but I'd still name this so its clear whats happening:
```suggestion
        attr_accessor :passed_examples
```